### DISCUSSION TITLE
Why we are not allowed to create polls in this discussions forum?

### DISCUSSION THREAD

**Discussion by ssbarnea** [community/community#42065] [Discussions] [answered]
### Select Topic Area

Question

### Body

Apparently even if we have more than 20 categories enabled on this forum, none of them allows us to create a poll type of post.

**Comment by DSDmark** [comment]
Hi @ssbarnea,

In my option, It is possible to create a poll using GitHub Issues. GitHub Issues are a way to track ideas, enhancements, tasks, and bugs for a project. You can use GitHub Issues to create a poll by creating an issue and asking a question with multiple choices as options.

**Comment by jfullstackdev** [comment]
you mean in this forum ? I was seeing last time several polls here but maybe it's banned now for non-Staff members here, not very sure of that

**Comment by jricciardi** [comment, marked as answer]
Hey there @ssbarnea - to enable polls, create a Discussion category and select the `Polls` category type. 

For more info, check out the official documentation here: https://docs.github.com/en/discussions/collaborating-with-your-community-using-discussions/participating-in-a-discussion#creating-a-poll
